pkg/aether_2_0_0/server/aether-2.0.0-server.go

```diff
@@ -39,6 +39,9 @@ type ServerInterface interface {
 	// POST /enterprises
 	// (POST /aether/v2.0.0/{target}/enterprises)
 	PostEnterprises(ctx echo.Context, target externalRef0.Target) error
+	// GET /enterprises/enterprise
+	// (GET /aether/v2.0.0/{target}/enterprises/enterprise)
+	GetEnterprisesEnterpriseList(ctx echo.Context, target externalRef0.Target) error
```

Comment:
also not immediately but could we generate `GetEnterprisesEnterpriseList` as `ListEnterprises`

Reply:
besides the names of these methods are generated by template `<Http-method><object>` and `List` is not a HTTP method. 
It would take some extra wrangling with the templates of `oapi-codegen` in places that I have not touched to change the name.
Since it's not externally facing, I'm not seeing a clear benefit to changing the name format? Is it for the App Gateway?

Reply:
I mean that makes sense from a yang perspective "just a container". I'm talking from purely readability